docs/source/cuda/cudapysupported.rst

```diff
@@ -36,6 +36,14 @@ The following Python constructs are not supported:
 The ``raise`` and ``assert`` statements are supported.
 See :ref:`nopython language support <pysupported-language>`.
 
+Printing of strings and integers is supported (see :ref:`nopython builtin
```

Comment:
I thought floats worked too? e.g.
```python
from numba import cuda
import numpy as np

@cuda.jit
def g(x):
    print(x[2])

z = np.arange(5.)
g[1, 1](z)

```

Reply:
Yes, they do - I should have put "numbers" rather than "integers" - now fixed. Many thanks for spotting!

Reply:
I'm not sure it can do complex domain?

Reply:
Doh! Another fix incoming...